corehq/apps/userreports/_design/views/data_sources_by_last_modified/map.js

```diff
@@ -0,0 +1,9 @@
+function (doc) {
+    if(doc.doc_type === "DataSourceConfiguration" && doc.is_deactivated == false) {
+        emit([
+            doc.last_modified,
+            doc.domain,
+            doc._id,
```

Comment:
Is `id` necessary in the key? I don't see it being used anywhere and it is also returned in the view results.

Reply:
yeah neither `domain` or  `_id` of these were necessary, but i like having the view results be readable which is why i put them in.